diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter2/kdoc/KdocComments.kt

```diff
@@ -306,7 +263,8 @@ class KdocComments(configRules: List<RulesConfig>) : DiktatRule(
 
             KDOC_NO_CONSTRUCTOR_PROPERTY.warnAndFix(configRules, emitWarn, isFixMode, warningText, node.startOffset, node) {
                 val classNode = node.parent { it.elementType == CLASS }!!
-                val newKdocText = if (isParamTagNeeded) "/**\n * @param $parameterName\n */" else "/**\n * @property $parameterName\n */"
+                val paramOrPropertyTagText = if (isParamTagNeeded) "@param" else "@property"
```

Comment:
Kotlin parser definitely has all these enum constants like `param` and `property` inside already. Use them.

Reply:
Currently, can't find these enum constants